src/Aquifer.Common/Utilities/BibleTextUtilities.cs

```diff
@@ -0,0 +1,42 @@
+using Aquifer.Common.Services.Caching;
+using Aquifer.Data;
+using Microsoft.EntityFrameworkCore;
+
+namespace Aquifer.Common.Utilities;
+
+public static class BibleTextUtilities
+{
+    public static async Task<IEnumerable<Chapter>> GetTextsByBibleId(int bookNumber, int chapter, int verse,
+        int bibleId, AquiferDbContext dbContext, ICachingVersificationService cachingService, CancellationToken ct)
+    {
+        var verseId = BibleUtilities.GetVerseId(bookNumber, chapter, verse);
+
+        var mappedVerseId = await cachingService.GetVersificationMapping(bibleId, verseId, ct);
+        var (mappedBookNumber, mappedChapter, mappedVerse) = BibleUtilities.TranslateVerseId(mappedVerseId);
+
+        var bibleTexts = await dbContext.BibleTexts
```

Comment:
I think you need a method that specifically handles transformations, because that may be all we care about. I think getting the transformations and getting texts are two separate things.

Reply:
makes sense. I'll split out the transformation part and have this call that